rego-templates/common/common.rego

```diff
@@ -29,3 +31,13 @@ with_default(obj, prop, default_value) = default_value{
 with_default(obj, prop, default_value) = obj[prop]{
  obj[prop]
 }
+
+severity_as_string(severity) := "Critical" if {
```

Comment:
This translation is the opposite,  it should be the following:
0 - unknown
1 - low
2 - medium
3 - high
4 - critical